extensions/wikia/FandomCreatorEmail/Controller/ContentUpdatedController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace FandomCreatorEmail\Controller;
+
+use FandomCreatorEmail\FandomCreatorEmailController;
+
+class ContentUpdatedController extends FandomCreatorEmailController {
+	/** @var string */
+	private $contentTitle;
+
+	/** @var string */
+	private $contentUrl;
+
+	public function initEmail() {
+		parent::initEmail();
+		$this->contentTitle = $this->request->getVal( 'contentTitle' );
+		$this->contentUrl = $this->request->getVal( 'contentUrl' );
+	}
+
+	public function getSubject(): string {
+		return $this->magicWordWrapper->wrap( function() {
+			return $this->getMessage(
+					'emailext-watchedpage-article-edited-subject',
+					$this->contentTitle,
+					$this->getCurrentUserName()
```

Comment:
If we call this internally from CGS -> MW will we have to pass along the `access_token` cookie? Or, can we load the user using the `X-Wikia-UserId`? Ah, nvm. It looks like this [has been done for us in the `EmailController`](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/EmailController.class.php#L463).